includes/wikia/services/VideoService.class.php

```diff
@@ -49,9 +52,21 @@ public function addVideo( $url ) {
 				$file = wfFindFile( $videoTitle );
 			}
 
-			// Add a default description if available and one doesn't already exist
-			$vHelper = new VideoHandlerHelper();
-			$vHelper->addDefaultVideoDescription( $file );
+			if ( !method_exists( $file, 'getTitle' ) ) {
```

Comment:
I would also add type hinting inside: addDefaultVideoDescription so it accepts only File as a parameter